package.json

```diff
@@ -58,8 +59,10 @@
     "jest": "jest",
     "lerna": "lerna",
     "lerna-prepare": "lerna run prepare",
-    "lint": "eslint --ext .js,.jsx packages/**/src",
+    "lint": "npm-run-all --continue-on-error --parallel lint:js lint:yaml",
```

Comment:
this will run the js and yaml lint tasks through to the end, show output for both and then bail with a non-zero exit code if either failed.